R/pkg/R/SQLContext.R

```diff
@@ -462,14 +462,19 @@ read.df <- function(sqlContext, path = NULL, source = NULL, ...) {
     source <- callJMethod(sqlContext, "getConf", "spark.sql.sources.default",
                           "org.apache.spark.sql.parquet")
   }
-  sdf <- callJMethod(sqlContext, "load", source, options)
```

Comment:
They are growing to over 30+ various variants of input/output functions. We now dump all of them in a namespace.